clarity/contracts/vaults-v2/README.txt

```diff
@@ -0,0 +1 @@
+liquidate vault, see https://explorer.hiro.so/txid/0x571ffa34bb0171dc275e9f1d1644c72d81e301267227c370426c861d3a161e0d?chain=mainnet
```

Comment:
REQ: could you remove this file pls?

Reply:
yes done in [2208a69](https://github.com/arkadiko-dao/arkadiko/pull/591/commits/2208a698685de75bb8715af6bc84e91711aaae9d)